### ISSUE TITLE
java.lang.ClassNotFoundException: can't load class io.dogboy.serializationisbad.core.ClassFilteringObjectInputStream

### ISSUE THREAD

**Issue by PeterStrick** [dogboy21/serializationisbad#67]
Trying to use serializationisbad-1.4.jar as an Java Agent in a 1.19.2 Fabric Server, crashes as soon as a Client either with or without the Java Agent Argument tries to login.

Modpack: All of Fabric 6
Fabric Loader: 0.14.22
`java -version`:
```
openjdk version "18.0.2-ea" 2022-07-19
OpenJDK Runtime Environment (build 18.0.2-ea+9-Ubuntu-222.04)
OpenJDK 64-Bit Server VM (build 18.0.2-ea+9-Ubuntu-222.04, mixed mode, sharing)
```
`uname -a`: `Linux uwu-server-ubuntu-3-2 5.15.0-1038-oracle #44-Ubuntu SMP Thu Jun 22 03:34:10 UTC 2023 aarch64 aarch64 aarch64 GNU/Linux`

Script used to start the Server:
```bash
#!/bin/bash
cd /mnt/Storage/AoF6
sudo java -javaagent:/mnt/Storage/serializationisbad-1.4.jar -Xmx4G -Xms4G -jar fabric.jar
```

Crash Report: [crash-2023-08-10_19.45.14-server.txt](https://github.com/dogboy21/serializationisbad/files/12315906/crash-2023-08-10_19.45.14-server.txt)

latest.log (with redacted IP): [latest.log](https://github.com/dogboy21/serializationisbad/files/12315940/latest.log)

serializationisbad-1.4.jar logs at the beginning: 
![image](https://github.com/dogboy21/serializationisbad/assets/60312421/b2f9d2a4-7c95-4776-9c2c-4f45aabd937b)

**Comment by dogboy21** [comment]
Thank you for reporting that issue!
Could you please test if this new version solves your problem? [serializationisbad-1.4.1.jar](https://ss.myftb.de/NGdqqdLA.zip)

**Comment by PeterStrick** [comment]
> Thank you for reporting that issue! Could you please test if this new version solves your problem? [serializationisbad-1.4.1.jar](https://ss.myftb.de/NGdqqdLA.zip)

Crashed again

[crash-2023-08-11_20.58.57-server.txt](https://github.com/dogboy21/serializationisbad/files/12324342/crash-2023-08-11_20.58.57-server.txt)

[latest.log](https://github.com/dogboy21/serializationisbad/files/12324343/latest.log)

**Comment by PeterStrick** [comment]
> Alright, I think I found the problem. Could you please try if this version solves your issue? [serializationisbad-1.4.1.jar](https://ss.myftb.de/z8EACoPS.zip)

Server seems to work perfectly with this Version

**Comment by dogboy21** [comment]
Alright, thank you for testing.
There will be a fix for this in the next SiB release.

**Comment by dogboy21** [comment]
Fixed in [version 1.5](https://github.com/dogboy21/serializationisbad/releases/tag/1.5)
